src/atoms/Layout/layout.module.scss

```diff
@@ -0,0 +1,96 @@
+@import 'src/assets/stylesheets/global';
+
+$gutter-width: 1.5rem;
+
+@mixin g-unit($cols: 1, $total-col-count: 12) {
```

Comment:
@jmcolella Do we want devs to be able to use the `g-unit` without using the `Layout` component?